components/brave_new_tab_ui/components/default/bitcoinDotCom/index.tsx

```diff
@@ -0,0 +1,300 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think it would probably make sense to have a new tab page component bundle separate for `bitcoinDotCom` but cc @petemill for a recommendation here.

Reply:
Do you mean a whole separate JS webpack output file that the page includes? I think it would be good to lazy-load _all_ the widgets. We can easily do this with [`import()` and `React.lazy`](https://reactjs.org/docs/code-splitting.html). Then we don't have to worry about impacting time-to-render as we add more widgets which is of course extremely important for this page. I am doing the same for the upcoming Brave Today feature.

Reply:
Per slack let's do this in a followup along with build a separate container